packages/ui/feature-dashboard/src/lib/pages/runs-table/runs-table.component.html

```diff
@@ -40,6 +40,16 @@
           </mat-option>
         </mat-select>
       </mat-form-field>
+      <mat-form-field>
+        <mat-label>Filter by date</mat-label>
+        <mat-date-range-input [formGroup]="dateFormGroup" [rangePicker]="runsPicker" [comparisonStart]="dateFormGroup.controls.start.value" 
+          [comparisonEnd]="dateFormGroup.controls.end.value">>
+          <input matStartDate placeholder="Start date" formControlName="start">
+          <input matEndDate placeholder="End date" formControlName="end">
```

Comment:
please add i18n-placeholder at the end of each tag and don't use formControlName instead use formControl and pass it the form controls from dateFormGroup.controls.start and dateFormGroup.controls.end

Reply:
[formControl]="dateFormGroup.controls.end"